### ISSUE TITLE
Proposal: add a "description" rel value

### ISSUE THREAD

**Issue by iherman** [w3c/epub-specs#1666]
More precisely: add a "description" term to the [Link Relationships Vocabulary](https://w3c.github.io/epub-specs/epub33/core/#app-link-vocab).

This would make it possible to use the [Link Element](https://w3c.github.io/epub-specs/epub33/core/#sec-link-elem) to refer to an extended description with a specific media type (e.g., HTML, Markdown...). 

For the origin of the use case see: #1650, the proposed solution in https://github.com/w3c/epub-specs/issues/1650#issuecomment-827781746. 

(Note, the [WG resolution](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-05-07-epub#resolution2) is not to accept the proposal as it was originally raised in #1650, this would be a replacement.)

**Comment by mattgarrish** [comment]
Do we need to mint another description value when you can already do:

```
<link rel="dcterms:description" href="..." media-type="..."/>
```

(or even `rel="dc:description"` or `rel="schema:description"`?)

Ideally, we're supposed to keep bare name rel values compatible the IANA and HTML link relations. Neither currently defines a description value. HTML has a DCTERMS.description proposed value, but I think in our case it would be better to keep the prefixed version.

**Comment by iherman** [comment]
Pragmatically, you are right, and maybe that is what we should do. My reticence comes from my semantic web influence, I guess: if I look at the meta and link elements and (informally, I know) translate them to proper triple statements, then what I get is that the statement with a property `dc:description` has a value of a URI and not a text which, I believe, is not in line with the DCMI specification. 

But I acknowledge that this is a theoretical purity issue; if everyone is fine with this approach, I am happy to settle on this.

**Comment by mattgarrish** [comment]
> then what I get is that the statement with a property `dc:description` has a value of a URI and not a text which

But isn't this what you're supposed to be doing with dcterms linked metadata? The [expected type is property](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/description), not literal text. I know they've relaxed their rules/expectations to allow text values all for the properties, but most are supposed to be links.

But even if we mint another description term, we're still going to end up with a triple whose value is a URL, no?

I'm fine adding an example to the spec if we want to acknowledge that you can use link for individual properties in another format and not just full records.

**Comment by iherman** [comment]
Heh. I did not realize that DCMI side-stepped this issue. I looked into the [dcterms RDF definition](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_terms.ttl), which also defined `dc:description` as an (RDF) subproperty of `dc:description` in the [dc RDF defintion](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dublin_core_elements.ttl) and, in both cases, all it says for the description (sic!) of `description` is:

> "Description may include but is not limited to: an abstract, a table of contents, a graphical representation, or a free-text account of the resource."

but there is no `rdfs:range` that would say the value is supposed to be a literal (which I expected would be the case). Ie, I was wrong in believing that we would be violating using `dc:description` as the property of a `<link>` element because we do require the value to be a URL.

Bottom line: using `dc:description` is indeed a good approach and solves the original use case and this issue is rubbish :-)

But adding an example that covers the original use case would actually be a good idea. I am happy to put in a PR on this on Monday, unless you beat me into it...
